src/Raven.Client/ServerWide/Tcp/TcpConnectionHeader.cs

```diff
@@ -28,7 +28,7 @@ public enum OperationTypes
         public static readonly int NumberOfRetriesForSendingTcpHeader = 2;
         public static readonly int ClusterTcpVersion = 10;
         public static readonly int HeartbeatsTcpVersion = 20;
-        public static readonly int ReplicationTcpVersion = 31;
+        public static readonly int ReplicationTcpVersion = 41;
```

Comment:
This will cause us to fail to talk to 4.0, sure.
Also will cause 4.0 to fail to talk to us.

I don't think that this is the desired behavior.